### DISCUSSION TITLE
Attributes

### DISCUSSION THREAD

**Discussion by AeryxSunyraX** [jrs0ul/nes_survival#50] [Q&A]
Where can i edit attributes or remove it?

**Comment by jrs0ul** [comment]
Attributes of what ? If it's color attributes, then it's last 64 bytes of a name-table. If it's a name-table that's being loaded, then you can modify color attributes in the editor, otherwise during the NMI you can write to the PPU starting at address $23C0, or $27C0
https://www.nesdev.org/wiki/PPU_attribute_tables

**Comment by AeryxSunyraX** [comment]
So the attributes is merged with the nametable then i must use the editor!

**Reply by jrs0ul** [reply]
It depends on what you are trying to achieve.
